src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/example-global-adf-cross-organization.yml

```diff
@@ -0,0 +1,341 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "This template is used for boostrapping accounts outside the current Organization"
+Parameters: #make sure before deploying this template that you change the parameters
+  DeploymentAccountId:
+    Description: Id of the deployment account 
+    Type: String
+    Default: deployment_account_id
+  KMSArn:
+    Description: Arn of the KMS key  
+    Type: String
+    Default: kms_arn
+  DeploymentAccountBucketName:
+    Description: Name of the S3 deployment bucket 
+    Type: String
+    Default: bucket_name
+
+Resources:
+  CodeCommitRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: "adf-codecommit-role"
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              AWS: !Sub arn:aws:iam::${DeploymentAccountId}:role/adf-codepipeline-role
+            Action:
+              - sts:AssumeRole
+          - Effect: Allow
+            Principal:
+              Service:
+                - events.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Path: /
+  CodeCommitPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: "adf-codecommit-role-policy"
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action:
+              - "codecommit:BatchGetRepositories"
+              - "codecommit:CancelUploadArchive"
+              - "codecommit:Get*"
+              - "codecommit:GitPull"
+              - "codecommit:List*"
+              - "codecommit:UploadArchive"
+              - "codepipeline:StartPipelineExecution"
+              - "events:PutEvents"
+            Resource: "*"
+          - Effect: Allow
+            Action:
+              - "s3:Get*"
+              - "s3:List*"
+              - "s3:Put*"
+            Resource:
+              - !Sub arn:aws:s3:::${DeploymentAccountBucketName}
+              - !Sub arn:aws:s3:::${DeploymentAccountBucketName}/*
+          - Effect: Allow
+            Action:
+              - "kms:Decrypt"
+              - "kms:Describe*"
+              - "kms:DescribeKey"
+              - "kms:Encrypt"
+              - "kms:GenerateDataKey*"
+              - "kms:Get*"
+              - "kms:List*"
+              - "kms:ReEncrypt*"
+            Resource: !Ref KMSArn
+      Roles:
+        - !Ref CodeCommitRole
+  CloudFormationPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: "adf-cloudformation-role-policy"
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Sid: "CloudFormation"
+            Action:
+              - cloudformation:*
```

Comment:
Could you put quotes around these, so all actions and other strings are quoted with double-quotes (")?